clients/v1_auth/v1_auth.go

```diff
@@ -0,0 +1,428 @@
+	for _, p := range auth.GetPermissions() {
+		ps = append(ps, permString(p))
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{
+		token: &v1Token{
+			ID:          auth.GetId(),
+			Description: auth.GetDescription(),
+			Token:       auth.GetToken(),
+			Status:      auth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type SetPasswordParams struct {
+	clients.AuthLookupParams
+	Password string
+}
+
+func (c Client) SetPassword(ctx context.Context, params *SetPasswordParams) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	auth, err := c.AuthorizationsApi.GetAuthorizationsID(ctx, id).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find authorization with User ID %q: %w", id, err)
+	}
+
+	password := params.Password
+	if password == "" {
+		password, err = c.StdIO.GetSecret("Please enter new password", 0)
+		if err != nil {
+			return err
+		}
+	}
+
+	if err := c.UsersApi.PostUsersIDPassword(ctx, auth.GetUserID()).Execute(); err != nil {
+		return fmt.Errorf("error setting password: %w", err)
+	}
+
+	return nil
+}
+
+func (c Client) printV1Tokens(params *v1PrintOpts) error {
+	if c.PrintAsJSON {
+		var v interface{}
+		if params.token != nil {
+			v = params.token
+		} else {
+			v = params.tokens
+		}
+		return c.PrintJSON(v)
+	}
+
+	headers := []string{
+		"ID",
+		"Description",
+		"Token",
+		"User Name",
+		"User ID",
+		"Permissions",
+	}
+	if params.deleted {
+		headers = append(headers, "Deleted")
+	}
+	if params.token != nil {
+		params.tokens = append(params.tokens, *params.token)
+	}
+
+	var rows []map[string]interface{}
+	for _, u := range params.tokens {
+		row := map[string]interface{}{
+			"ID":          u.ID,
+			"Description": u.Description,
+			"Token":       u.Token,
+			"User Name":   u.UserName,
+			"User ID":     u.UserID,
+			"Permissions": u.Permissions,
+		}
+		if params.deleted {
+			row["Deleted"] = true
+		}
+		rows = append(rows, row)
+	}
+	return c.PrintTable(headers, rows...)
+}
+
+func permString(p api.Permission) string {
+	ret := p.GetAction() + ":"
+	r := p.GetResource()
+
+	if r.GetOrgID() != "" {
+		ret += "orgs" + r.GetOrgID()
+	}
+	ret += r.GetType()
+	if r.GetId() != "" {
+		ret += r.GetId()
+	}
+	return ret
+}
+
+func (c Client) getAuthReqID(ctx context.Context, params *clients.AuthLookupParams) (id string, err error) {
+	if params.ID.Valid() {
+		id = params.ID.String()
+	} else {
+		userid, err := c.UsersApi.GetUsers(ctx).Name(params.Username).Execute()
+		if err != nil || len(userid.GetUsers()) == 0 {
+			err = fmt.Errorf("could not find User ID from username %q: %w", params.Username, err)
+		} else {
+			id = userid.GetUsers()[0].GetId()
+		}
+	}
+	return
+}
+
+func (c Client) getOrgID(ctx context.Context, params clients.OrgParams) (string, error) {
+	if params.OrgID.Valid() || params.OrgName != "" || c.ActiveConfig.Org != "" {
```

Comment:
(Sorry if I haven't caught it in past PRs)

More conventional Go would be to:
1. Invert the conditional here
2. Return the error as the body
3. Leave the remaining logic un-indented after the `if`

[For example](https://github.com/influxdata/influx-cli/blob/main/clients/bucket/create.go#L24-L28)